packages/client/src/declarations/index.ts

```diff
@@ -1920,6 +1920,18 @@ export const declarationsReducer: LoopReducer<IDeclarationsState, Action> = (
           )
         )
       }
+      if (declarationNextToUnassign) {
```

Comment:
`declarationNextToUnassign` could probably called something else to make it more descriptive. Could you add a comment here to clarify the logic please? I think I'm not following 😅